scapy/layers/rtp.py

```diff
@@ -7,34 +7,51 @@
 RTP (Real-time Transport Protocol).
 """
 
-from scapy.packet import *
 from scapy.fields import *
+from scapy.packet import *
 
 _rtp_payload_types = {
     # http://www.iana.org/assignments/rtp-parameters
-    0:  'G.711 PCMU',    3:  'GSM',
-    4:  'G723',          5:  'DVI4',
-    6:  'DVI4',          7:  'LPC',
-    8:  'PCMA',          9:  'G722',
-    10: 'L16',           11: 'L16',
-    12: 'QCELP',         13: 'CN',
-    14: 'MPA',           15: 'G728',
-    16: 'DVI4',          17: 'DVI4',
-    18: 'G729',          25: 'CelB',
-    26: 'JPEG',          28: 'nv',
-    31: 'H261',          32: 'MPV',
-    33: 'MP2T',          34: 'H263' }
+    0: 'G.711 PCMU', 3: 'GSM',
```

Comment:
Hi! I understand you want to change this, but please don't. To avoid breaking the code history, please only change what you need to change. Thanks!